Tools/Scripts/update-webkitgtk-libs

```diff
@@ -24,6 +24,8 @@ use webkitdirs;
 my $scriptsDir = relativeScriptsDir();
 if (defined $ENV{'WEBKIT_JHBUILD'} and $ENV{'WEBKIT_JHBUILD'}) {
     system("perl", "$scriptsDir/update-webkit-libs-jhbuild", "--gtk", @ARGV) == 0 or die $!;
+} elsif (defined $ENV{'WEBKIT_CROSS_TARGET'} or grep(/^--cross-target/, @ARGV)) {
+    system("perl", "$scriptsDir/cross-toolchain-helper", "--build-toolchain", @ARGV) == 0 or die $!;
```

Comment:
Shouldn't this be `"python3"`?

Reply:
ups, yes.. :sweat_smile: 
good catch, i will fix it

Reply:
Funny enough this worked.
It seems that if you run `perl something` and that something has a shebang, then perl will execute it like if you ran it from a shell... TIL :teacher: